library/X509/Test/BaseTestCase.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
We don't need that. Just extend `PHPUnit\Framework\TestCase` in concrete test cases.